nrf_wifi/fw_if/umac_if/inc/default/fmac_api.h

```diff
@@ -986,6 +986,32 @@ enum nrf_wifi_status nrf_wifi_fmac_set_listen_interval(void *fmac_dev_ctx,
 enum nrf_wifi_status nrf_wifi_fmac_set_ps_wakeup_mode(void *fmac_dev_ctx,
 							unsigned char if_idx,
 							bool ps_wakeup_mode);
+
```

Comment:
Should this be protected by `#ifdef CONFIG_NRF700X_RAWDATA_TX` ?

Reply:
done